.dockerignore

```diff
@@ -0,0 +1,6 @@
+.pytest_cache
+.aws_sam
+__pycache__
+**/__pycache__
+.vscode
+tests
```

Comment:
should we ignore `events` as well?
Probably `venv` as well

We can add something like this here:
```
__pycache__
**/__pycache__
events
*.pyc
*.pyo
*.pyd
.Python
venv
env
pip-log.txt
pip-delete-this-directory.txt
.tox
.coverage
.coverage.*
.cache
nosetests.xml
coverage.xml
*.cover
*.log
.git
.mypy_cache
.pytest_cache
.hypothesis
```